.circleci/config.yml

```diff
@@ -348,19 +350,19 @@ workflows:
             parameters:
               instance-type: [ edge ]
           requires:
-            - update-dependencies-8 << matrix.instance-type >>
+            - update-dependencies-9 << matrix.instance-type >>
       - run-d9-check:
           name: run-d9-check
           matrix:
             parameters:
               instance-type: [ edge ]
           requires:
-            - update-dependencies-8 << matrix.instance-type >>
-      - run-code-coverage:
-          name: run-code-coverage
-          matrix:
-            parameters:
-              instance-type: [ edge ]
-          requires:
-            - update-dependencies-8 << matrix.instance-type >>
-            - run-unit-kernel-tests-8 << matrix.instance-type >>
+            - update-dependencies-9 << matrix.instance-type >>
+#      - run-code-coverage:
```

Comment:
Why is this commented?

Reply:
Circleci config file had code coverage job but never collected the code coverage because of incorrect code-coverage config.
In CicleCI test it always showed this error
```Fatal error: Cannot declare class MigrateBlockedIpsTest, because the name is already in use``` ,
and now the code coverage implementation with codecov is in progress, once its done we will un-comment code coverage from config file.

Reply:
We agreed that this will be enabled in the next PR